webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -500,40 +500,40 @@ const ModelRegistry: React.FC = () => {
         </Space>
       )}
       title="Model Registry">
-      {(models.length === 0 && !isLoading && filterCount === 0) ?
-        (
-          <div className={css.emptyBase}>
-            <div className={css.icon}>
-              <Icon name="model" size="mega" />
-            </div>
-            <h4>No Models Registered</h4>
-            <p className={css.description}>
-              Track important checkpoints and versions from your experiments.&nbsp;
-              <Link external path={paths.docs('/post-training/model-registry.html')}>
-                Learn more
-              </Link>
-            </p>
+      {(models.length === 0 && !isLoading && filterCount === 0) ? (
```

Comment:
nit avoid to avoid introducing the changes

Reply:
I'll revert this. I'm hoping to find some more JSX eslint rules to auto arrange these to have less variance.